src/modules/form/data/service.json

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
Is this used for a story or a mock? If so can you move it into the `test/__mocks__` dir

Reply:
Used for a story so I could test different form defs. I'll probably remove most of the ones I added since I don't need them beyond development